src/com/owncloud/android/ui/activity/FileDisplayActivity.java

```diff
@@ -144,7 +144,7 @@
     private boolean mSyncInProgress = false;
 
     private static String DIALOG_UNTRUSTED_CERT = "DIALOG_UNTRUSTED_CERT";
-    private static String DIALOG_CREATE_FOLDER = "DIALOG_CREATE_FOLDER";
+    public static String DIALOG_CREATE_FOLDER = "DIALOG_CREATE_FOLDER";
```

Comment:
You also created public ```FileDisplayActivity#createFolder() ```; could ```OCFileListFragment``` call it instead of directly creating a dialog? It's duplicated code. That way the constant may stay private also.